streams/src/main/java/org/apache/kafka/streams/processor/internals/ProcessorContextImpl.java

```diff
@@ -103,7 +102,6 @@ public ProcessorStateManager stateManager() {
     @Override
     public void register(final StateStore store,
                          final StateRestoreCallback stateRestoreCallback) {
-        storeToChangelogTopic.put(store.name(), ProcessorStateManager.storeChangelogTopic(applicationId(), store.name()));
         super.register(store, stateRestoreCallback);
```

Comment:
If we only pass through, we can remove the overwrite of the method completely.

Reply:
Ack